src/EnduranceTests/TestUtils/TestEnvironment.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.AzureServiceBus.EnduranceTests.TestUtils
+{
+    using System;
+    using Microsoft.WindowsAzure.Storage;
+
+    internal static class TestEnvironment
+    {
+        public static string AzureServiceBus
+        {
+            get
+            {
+                var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString");
```

Comment:
I used the same convention the other tests used for the connection string to make running this in other contexts easier.